### PR TITLE
Add new committer info

### PR DESCRIPTION
**Pull request by totalo**

### CODE REVIEW

**Comment by RaigorJiang** [comment]
Congratulations @totalo

**Comment by totalo** [comment]
> Congratulations @totalo

Thank you.

File: docs/community/content/team/_index.cn.md [hunk 1]
```diff
@@ -30,29 +30,30 @@ chapter = true
 
 ### 提交者
 
-| 名字   | GitHub ID   |
-| :---- | :---------- |
```
Comment (sandynz) [inline docs/community/content/team/_index.cn.md:34]: It's better to just add new committer info, just as the title says. Formatting could be a new PR.

**Review by sandynz — CHANGES_REQUESTED** [review]

File: docs/community/content/team/_index.cn.md (hunk 1 above)
Comment (totalo) [reply docs/community/content/team/_index.cn.md:34]: All right.

**Review by sandynz — APPROVED** [review]

**Pull request merged**